src/registrar/management/commands/load_transition_domain.py

```diff
@@ -312,8 +458,10 @@ def handle(  # noqa: C901
 
         # Start parsing the main file and create TransitionDomain objects
         logger.info("Reading domain-contacts data file %s", domain_contacts_filename)
+        total_lines = TerminalHelper.get_file_line_count(domain_contacts_filename)
         with open(domain_contacts_filename, "r") as domain_contacts_file:
             for row in csv.reader(domain_contacts_file, delimiter=sep):
+                # TerminalHelper.printProgressBar(total_rows_parsed, total_lines)
```

Comment:
**question**: was this something just being tried out?

Reply:
Yeah I believe so